package.json

```diff
@@ -33,42 +43,48 @@
   "homepage": "https://github.com/apollostack/apollo-proxy#readme",
   "dependencies": {
     "lodash.uniq": "^4.3.0",
-    "node-uuid": "^1.4.7"
+    "node-uuid": "^1.4.7",
+    "graphql-typings": "git://github.com/DxCx/graphql-typings.git#tmp-pkg"
```

Comment:
looks like @nitintulani just merged the patches, so let's just use his npm package =)